frontend/src/pages/ApplicationsPage.tsx

```diff
@@ -74,7 +76,7 @@ const ApplicationsPage: React.FC<ApplicationsPageProps> = ({
 
   const renderContents = () => {
     if (loadError) {
-      return (
+      return !errorStatePage ? (
```

Comment:
Not sure if `errorStatePage` should be checked as a part of `loadError`, but apart from everything else looks fine, including the button navigation for "See all pipelines".
<img width="1287" alt="Screenshot 2023-07-13 at 3 38 46 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/faeb8d33-22ec-44fb-b6e7-a6b2222b1f6d">

Reply:
So this pipelineLoadError is related to the loadError  , so I have put this condition here , we have a another scenario for this https://github.com/opendatahub-io/odh-dashboard/pull/1497/files#diff-60ce67b4d05babd2f2fa1532ce4290a55e1466f2c59b71bc654e1d6ae78498d9L103 , I referred this one

Reply:
This link leads to your own changes

Reply:
Not sure I'm a fan of this -- I get what you're after, but it adds complexity to the ApplicationPage for an existing scenario that already exists. 🤔 It either is an error or we need to support a 404 as a special use-case. Having everyone provide their own potential component will just open us up to a wide variety of ways this is done and make it harder to be consistent.